### PR TITLE
Add Account: Update Supported Options + FindETH [ch4445]

### PR DESCRIPTION
**Pull request by GuillaumeRx**
- [x] change supported account order

- [x] add missing icons

- [X] add findeth.io link

- [X] correct panels size

[Story details](https://app.clubhouse.io/mycrypto/story/4445/add-account-update-supported-options-findeth-link)

### CODE REVIEW

File: common/v2/components/WalletUnlock/DeterministicWallets.tsx [hunk 1]
```diff
@@ -330,6 +344,12 @@ export function DeterministicWalletsClass({
           {translate('ACTION_6')}
         </Button>
       </div>
+      <FooterLink>
+        <Typography>Can't find your ETH ?</Typography>&nbsp;
```
Comment (FrederikBolding) [inline common/v2/components/WalletUnlock/DeterministicWallets.tsx]: Use translation for this!

File: common/v2/features/AddAccount/stories.tsx [hunk 2]
```diff
@@ -23,47 +21,38 @@ export const IS_NOT_ELECTRON_AND_IS_NOT_DEV: boolean = !IS_ELECTRON && !IS_DEV;
 
 export const getStories = (): IStory[] => [
   {
-    name: WalletId.WEB3,
```
Comment (FrederikBolding) [inline common/v2/features/AddAccount/stories.tsx]: Why is the ordering changed here?

File: common/v2/components/WalletList/WalletList.tsx [hunk 3]
```diff
@@ -39,7 +39,7 @@ const Description = styled.p`
 const WalletsContainer = styled.div`
   display: flex;
   justify-content: center;
-  flex-wrap: wrap;
+  flex-wrap: wrap-reverse;
```
Comment (FrederikBolding) [inline common/v2/components/WalletList/WalletList.tsx]: According to the card and Luker the wallets should be in the following rows:

MM, WC
Ledger, Trezor
Keystore, Private Key, mnem
View only

File: common/v2/components/WalletUnlock/DeterministicWallets.tsx [hunk 4]
```diff
@@ -26,6 +28,18 @@ import { DeterministicWalletData, getDeterministicWallets } from 'v2/services/Wa
...
 
+const FooterLink = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: baseline;
+  p {
+    font-size: 16px;
+  }
+  a {
+    color: ${COLORS.BLUE_BRIGHT};
```
Comment (FrederikBolding) [inline common/v2/components/WalletUnlock/DeterministicWallets.tsx]: When using the translation you can use Markdown, so we don't have to do the extra styling here.

File: common/v2/features/AddAccount/stories.tsx (hunk 2 above)
Comment (GuillaumeRx) [reply common/v2/features/AddAccount/stories.tsx]: In order to have 2-3-3 I had to set `flex-wrap` to `wrap-reverse` so I needed the list to be reversed

File: common/v2/components/WalletList/WalletList.tsx (hunk 3 above)
Comment (GuillaumeRx) [reply common/v2/components/WalletList/WalletList.tsx]: Isn’t it the case ?

File: common/v2/components/WalletUnlock/DeterministicWallets.tsx (hunk 4 above)
Comment (GuillaumeRx) [reply common/v2/components/WalletUnlock/DeterministicWallets.tsx]: Nice tip !

Comment (GuillaumeRx) [reply common/v2/components/WalletUnlock/DeterministicWallets.tsx]: But you can't change color with MD ? 🤔

Comment (FrederikBolding) [reply common/v2/components/WalletUnlock/DeterministicWallets.tsx]: It'll use standard styling for links, like we use on the rest of the site.

File: common/v2/features/AddAccount/stories.tsx (hunk 2 above)
Comment (FrederikBolding) [reply common/v2/features/AddAccount/stories.tsx]: It should be 2-2-3-1, like I said in my other comment 😄

File: common/v2/components/WalletList/WalletList.tsx (hunk 3 above)
Comment (FrederikBolding) [reply common/v2/components/WalletList/WalletList.tsx]: No, it should be 2-2-3-1, you have 2-3-3 😄

File: common/v2/components/WalletUnlock/DeterministicWallets.tsx [hunk 5]
```diff
@@ -330,6 +344,12 @@ export function DeterministicWalletsClass({
           {translate('ACTION_6')}
         </Button>
       </div>
+      <FooterLink>
+        <Typography>{translateRaw('FIND_ETH_PHRASE')}</Typography>&nbsp;
+        <Link href="https://findeth.io" target="_blank" rel="noreferrer">
```
Comment (FrederikBolding) [inline common/v2/components/WalletUnlock/DeterministicWallets.tsx]: Please use Markdown in the string 😄

**Pull request merged**